lib/fpm/package/deb.rb

```diff
@@ -905,7 +905,7 @@ def mkdir_p(dir)
   def to_s(format=nil)
     # Default format if nil
     # git_1.7.9.3-1_amd64.deb
-    return super("NAME_FULLVERSION_ARCH.TYPE") if format.nil?
+    return super("NAME_FULLVERSION_ARCH.EXTENSION") if format.nil?
```

Comment:
I wonder why ruby people don't use the [ternary if](http://docs.ruby-lang.org/en/2.1.0/syntax/control_expressions_rdoc.html#label-Ternary+if) control expression? This line and line below could be simply reduced to:
```
super(format.nil? "NAME_FULLVERSION_ARCH.EXTENSION" : format)
```
Similarly in other modules.